### DISCUSSION TITLE
[Calendar module] formatter:{} setting not working properly when the Type: setting is set to "datetime"

### DISCUSSION THREAD

**Discussion by cheesyMan** [fomantic/Fomantic-UI#1645] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report
**When customizing the date format in **_Calendar_** module, hour:minutes are always showed, even if the formatter:{} function returns nothing.**

## Steps to reproduce
1. Create a Calendar element with _type: "datetime"_ as initial setting
2. Customize the date format via the **formatter:{}** setting. Let the formatter function return "".  
3. The calendar field shows hour/minute anyway.

## Expected result
The calendar field should show nothing

## Actual result
Date/Hour of selected date

## Testcase
https://jsfiddle.net/6Ltz0ncv/3/

## Screenshot (when possible)
![Snipaste_2020-08-18_12-59-36](https://user-images.githubusercontent.com/55018265/90505390-d37d6600-e152-11ea-8e2d-05a0edc3801b.jpg)
)

## Version
2.8.6

**Comment by lubber-de** [comment, marked as answer]
You are using the wrong formatter: You used `date` formatter, but `datetime` as type.
If you are using `datetime` as formatter, it works as expected
See your adjusted jsfiddle here https://jsfiddle.net/lubber/9sq3zryL/1/

**Comment by cheesyMan** [comment]
> You are using the wrong formatter: You used `date` formatter, but `datetime` as type.
> If you are using `datetime` as fomatter, it works as expected
> See your adjusted jsfiddle here https://jsfiddle.net/lubber/9sq3zryL/1/

Really stupid error. Thanks a lot! 👍
